_data/projects/auth-jwt.yml

```diff
@@ -0,0 +1,13 @@
+---
+name: auth-jwt
+desc: 'A demo to learn JWT by reverse engineering'
+site: https://github.com/gitcommitshow/auth-jwt
+tags:
+- jwt
+- authentication
+- learning
+- rfc-7519
+- express
+upforgrabs:
+  name: good first issue
+  link: https://github.com/gitcommitshow/auth-jwt/labels/good%20first%20issue
```

Comment:
You don't have any issues marked with this label, at least mark few issues so that users can visit and work on that.

Reply:
Added [issue with `good first issue`](https://github.com/gitcommitshow/auth-jwt/issues/6) tag. Will keep adding more.
Is it good to go now?

Reply:
Yep